Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:stretch as builder
```

Comment:
What options besides `golang:stretch` are available? Does this one make the most sense? Will this need to be updated or changed in the future? Just trying to learn and understand! Thanks!

Reply:
there are also alpine and microsoft options available. but there is no buster available right now. 
it would be possible to use golang:1.12 or golang:1 to keep up with the docker tags of golang

Reply:
I imagine we can massage this in the future if it breaks.